### DISCUSSION TITLE
Welcome to Quasar Discussions!

### DISCUSSION THREAD

**Discussion by rstoenescu** [quasarframework/quasar#8679] [Announcements]
## 👋 Welcome!
 We’re using Discussions now as a place to connect with other members of our community. 

 We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This will be our new community discussion home (along with our [Discord Server](https://discord.gg/CeFeQPpC)). The old Forum will be open for about a week for new topics. After that week (approx. 2nd of April), it will be set to "read only" and turned off at the end of the year (around 2021/12/28).

**Comment by matejmohar** [comment]
I very much like this change, but would it be possible to have some archive of the forum online? As there are still some questions/answers that provide useful information and if the forum will shut down, lots of that will be lost.

**Reply by hawkeye64** [reply]
I believe the forum will still exist in read-only mode so it can be searchable. But, it will no longer be a place where questions will be answered.

**Reply by matejmohar** [reply]
Then it's good. I understood before that forum will be shut down at the end of the year and in that case, not accessible anymore.
